package.json

```diff
@@ -17,14 +18,21 @@
   "devDependencies": {
     "@swc/cli": "^0.1.46",
     "@swc/core": "^1.2.66",
+    "@testing-library/jest-dom": "^5.14.1",
+    "@testing-library/react": "^12.0.0",
+    "@types/jest": "^27.0.1",
+    "@types/node": "^16.6.1",
     "@types/react": "^17.0.14",
     "@typescript-eslint/eslint-plugin": "^4.28.4",
     "@typescript-eslint/parser": "^4.28.4",
     "eslint": "^7.31.0",
     "eslint-config-google": "^0.14.0",
+    "eslint-plugin-jest-dom": "^3.9.0",
     "eslint-plugin-react": "^7.24.0",
+    "jest": "^27.0.6",
     "lerna": "^3.22.1",
     "swc-loader": "^0.1.14",
+    "ts-jest": "^27.0.4",
```

Comment:
I put all this in the root, and this means I can run (on my machine!!!) `yarn test` anywhere and all the tests run.. maybe we want more finer grained control to just run tests for a particular package, but there are jest cli options for that type of thing anyway 🤔

Reply:
yeah, agree. certainly at this stage having them in the root is best